education.html

```diff
@@ -5,8 +5,10 @@
     <title>John Doe | Education</title>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="keywords" content="free portfolio template,portfolio template, portfolio website template, open source, open sourcing, contributing to open source" />
-    <meta name="description" content="This project serves free and beautiful portfolio template for everyone. Anyone can showcase their work including moocs, education, projects, experience,etc. to the world." />
+    <meta name="keywords"
+        content="free portfolio template,portfolio template, portfolio website template, open source, open sourcing, contributing to open source" />
```

Comment:
Please undo all indentation and linting related changes.

Reply:
Nothing is changing in code.
only changed in "format document"
and it's work.

Reply:
I am aware of that 😅. Irrespective of that these sections will be counted as changed in a commit and will give rise to merge conflict for other PRs